frontend/src/static/js/v_summary.js

```diff
@@ -268,6 +268,7 @@ window.vSummary = {
     // model functions //
     updateFilterSearch(evt) {
       this.filterSearch = evt.target.value;
+      this.getFiltered();
```

Comment:
Maybe we can watch on `filterSearch` just like other flags?

Reply:
if we create a watcher for `filterSearch`, the watcher will be triggered every time u type/modify a character, which is not ideal for our case. We want to trigger `getFiltered()` only when the user presses the key `enter` after keying the search value.

Reply:
Reasonable。